Makefile

```diff
@@ -24,8 +26,67 @@ release:
 mrconfig:
 	./scripts/update_mrconfig.py
 
+muconfig:
+	mu group add salt-formulas --empty
+	mu register $(FORKED_FORMULAS_DIR)/*
+
 html:
 	make -C doc html
 
 pdf:
 	make -C doc latexpdf
+
+FORKED_FORMULAS_DIR=fformulas
+FORMULAS=`python3 -c 'import sys; sys.path.append("scripts");from update_mrconfig import *; print(*get_org_repos(make_github_agent(), "salt-formulas"), sep="\n")'| egrep 'salt-formula-' | sed 's/salt-formula-//'`
+
+scripts_prerequisites:
+	pip3 install -r scripts/requirements.txt
+
+list: scripts_prerequisites
+	@echo $(FORMULAS)
+
+update_forks: scripts_prerequisites
```

Comment:
You don't want to clone your forks into different directory, that consumes disk space and doesn't make any sense. You want to use remote pointing to your fork instead.